scss/components/_radio.scss

```diff
@@ -17,13 +17,15 @@
 
           @media screen and (-ms-high-contrast: active) {
             border-color: Highlight;
+            background-image: url("data:image/svg+xml;charset=utf8,<svg xmlns='http://www.w3.org/2000/svg' viewBox='-4 -4 8 8'><circle r='4' stroke='highlight' stroke-width='1.5' /></svg>");
```

Comment:
@geppy border-image is not working while adding an SVG in high contrast mode, so used `background-image` with stroke instead of fill.